### DISCUSSION TITLE
[FEATURE] challenge accepted

### DISCUSSION THREAD

**Discussion by DenisYaroshevskiy** [jfalcou/eve#1553] [Research & Development]
![image](https://user-images.githubusercontent.com/11256077/136690080-4adbd5dc-06ca-4725-ad53-cbae030c824c.png)

We don't have `partition`, `sort` or `iota` but it's a nice goal.

**Comment by jfalcou** [comment]
iota should be easy enough as it's basically just filling a range with [0 1 2 ...N] + offset
sort & partition : https://hal.inria.fr/hal-01512970v1/document

**Comment by DenisYaroshevskiy** [comment]
I've seen this one on sort: https://bits.houmus.org/2020-01-28/this-goes-to-eleven-pt1

iota should be easy enough yes.

**Comment by DenisYaroshevskiy** [comment]
Reply here https://twitter.com/code_report/status/1447005830025449472?t=Xz9zjQD7KJO8QAR2EN9yBg&s=19
